src/Newsletter.php

```diff
@@ -241,6 +241,38 @@ public function updateContent(string $campaignId, string $html, array $options =
         return $response;
     }
 
+    public function getListInterestCategories(int $count = 10, int $offset = 0, string $type = null, string $listName = '')
```

Comment:
I would rather set default limit to higher value, like 100 at least.
Interest category data is tiny there is no risk of memory overflow, while usually developer wishes to fetch them all.

Reply:
I just pushed up a commit with the count increased to 100